pkg/synthetictests/duplicated_events.go

```diff
@@ -100,14 +96,23 @@ var knownEventsBugs = []knownProblem{
 		Regexp: regexp.MustCompile(`ns/openshift-network-diagnostics pod/network-check-target-[a-z0-9]+ node/[a-z0-9.-]+ - reason/NetworkNotReady network is not ready: container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: No CNI configuration file in /etc/kubernetes/cni/net\.d/\. Has your network provider started\?`),
 		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1986370",
 	},
-	{
-		Regexp: regexp.MustCompile(`ns/openshift-machine-api machine/.* - reason/Update Updated Machine .*`),
-		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1988992",
-	},
 	{
 		Regexp: regexp.MustCompile(`ns/.* service/.* - reason/FailedToDeleteOVNLoadBalancer .*`),
 		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1990631",
 	},
+	{
+		Regexp: regexp.MustCompile(`ns/.*horizontalpodautoscaler.*failed to get cpu utilization: unable to get metrics for resource cpu: no metrics returned from resource metrics API.*`),
+		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1993985",
+	},
+	{
+		Regexp: regexp.MustCompile(`ns/.*unable to ensure pod container exists: failed to create container.*slice already exists.*`),
+		BZ:     "https://bugzilla.redhat.com/show_bug.cgi?id=1993980",
+	},
+	{
+		Regexp: regexp.MustCompile(`ns/openshift-etcd-operator deployment/etcd-operator - reason/UnhealthyEtcdMember unhealthy member.*`),
```

Comment:
is this happening outside of upgrade jobs?

Reply:
No and I see you already handled this in https://github.com/openshift/origin/commit/de8df4de72a40bc6bc561dc9562d49d50ec45e54

I've removed it